resources/migrations/001_update_migrations.yaml

```diff
@@ -3394,35 +3394,48 @@ databaseChangeLog:
 
   - changeSet:
       id: v48.00-008
-      validCheckSum: 8:c0a512701d6dfd5cc1c4a689919be074
+      validCheckSum: ANY
```

Comment:
I used any because we had a lot of fun with liquibase changing checksum already and I don't want to deal with it anymore.
see this [thread](https://metaboat.slack.com/archives/C052ZBWRG3W/p1713588967744239?thread_ts=1713537103.654069&cid=C052ZBWRG3W)

Reply:
on my machine I got this check sum`9:ad3f69be24960003ce545c16ac7922eb`. it prefixes with `9:` not `8:` so I feel a bit hesitant to write it in.